template-processing/handlebars/gradle.properties

```diff
@@ -1,4 +1,4 @@
 version=2.0.0-SNAPSHOT
-knotx.version=2.0.0
+knotx.version=2.1.1-SNAPSHOT
```

Comment:
ToDo: change when 2.2 released. 
The idea of examples it to not force anyone to build the whole Knot.x stack (and SNAPSHOTS are not publicly available).